src/renderer/components/CustomIcons.tsx

```diff
@@ -139,7 +139,29 @@ export const IconFactory = memo(
                     />
                 );
             default:
-            // nothing
+                break;
+        }
+
+        // using segmenter to account for non-latin and emoji characters
+        const isSingleCharacter = [...new Intl.Segmenter().segment(icon)].length === 1;
+        if (isSingleCharacter) {
+            return (
+                <h5
+                    aria-hidden="true"
+                    className="chakra-heading"
+                    role="presentation"
+                    style={{
+                        color: accentColor,
+                        'font-weight': 'bold',
+                        'text-align': 'center',
+                        'vertical-align': 'middle',
+                        'text-rendering': 'geometricPrecision',
+                        'font-family': 'Noto Emoji, Open Sans, sans-serif',
```

Comment:
```suggestion
                        fontWeight: 'bold',
                        textAlign: 'center',
                        verticalAlign: 'middle',
                        textRendering: 'geometricPrecision',
                        fontFamily: 'Noto Emoji, Open Sans, sans-serif',
```
React's style objects use camel case instead of kebab kase